src/lib/components/BucketAggregation/BucketAggregationValues.js

```diff
@@ -19,6 +19,7 @@ class BucketAggregationValues extends Component {
     this.aggName = props.aggName;
     this.childAgg = props.childAgg;
     this.onFilterClicked = props.onFilterClicked;
+    this.parentValue = props.parentValue;
```

Comment:
I have the feeling that you might have to change the cmp `BucketAggregation`, to pass the initial value for this prop.

Reply:
We are going for a different solution, where we don't specify any more the id of the element. There is no need to pass the parent anymore